todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/source/TasksDataSource.kt

```diff
@@ -29,39 +29,15 @@ import com.example.android.architecture.blueprints.todoapp.data.Task
  */
 interface TasksDataSource {
```

Comment:
I decided to make all functions in`TasksDataSource` suspended. This is how coroutines are designed to work. (Indicate that function may do intensive operation.) 

See: https://github.com/dmytrodanylyk/android-architecture/blob/dev-todo-mvp-kotlin-coroutines/todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/source/TasksDataSource.kt

(Callbacks to be removed)